lib/features/guide_detail_view/utils/get_the_latest_date.dart

```diff
@@ -0,0 +1,30 @@
+import "package:flutter/material.dart";
+import "package:intl/intl.dart";
+
+import "../repository/getGuideDetails.graphql.dart";
+
+extension GetTheLatestDateGuide on BuildContext {
+  String? getTheLatesCreatedDateGuide({
+    List<Query$GetGuideDetails$FAQ_Types_by_id$questions?>? questions,
+  }) {
+    final DateTime? newestDate = questions
+        ?.where((e) => e?.FAQ_id?.date_created != null)
+        .map((e) => e!.FAQ_id!.date_created!)
+        .reduce((a, b) => a.isAfter(b) ? a : b);
```

Comment:
I'm not sure, but I recall we have some extension that quite nicely filters nulls. It is called where not null or sth like that.
I'm not sure thought if it'd fit in this scenario but def worth checking ;)